ansible/roles/products/common/hand_config/tasks/main.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Create temporary script file
+  file:
+    path: /tmp/update_hand_config.sh
+    state: touch
+    mode: u=rwx,g=rwx,o=rwx
+
+- name: Add content to script file
+  copy:
+    dest: /tmp/update_hand_config.sh
+    content: |
+      #!/usr/bin/env bash
+      # Used to update hand config.
+      cd ~/projects/shadow_robot/base_deps/src/sr_hand_config
+      git_branch=$(git rev-parse --abbrev-ref HEAD | xargs)
+      if [[ `git status --porcelain` ]]; then
+        file_changed=true
+      else
+        file_changed=false
+      fi
```

Comment:
if I have my changes on a branch, changes committed and pushed, this will fail as I am not on master.